formats/hocon/src/main/kotlin/kotlinx/serialization/hocon/PackageMigrations.kt

```diff
@@ -4,12 +4,12 @@
 
 package org.jetbrains.kotlinx.serialization.config
 
-
 private const val pkg = "kotlinx.serialization.config"
 private const val message = "Moved to $pkg package"
 
+@Suppress("DEPRECATION_ERROR")
 @Deprecated(message, ReplaceWith("ConfigParserConfiguration", pkg), level = DeprecationLevel.ERROR)
 typealias ConfigParserConfiguration = kotlinx.serialization.config.ConfigParserConfiguration
 
 @Deprecated(message, ReplaceWith("ConfigParser", pkg), level = DeprecationLevel.ERROR)
```

Comment:
I think the `ReplaceWith` expression needs to be updated as well.
```suggestion
@Deprecated(message, ReplaceWith("Hocon", pkg), level = DeprecationLevel.ERROR)
```

Reply:
Indeed, these migrations aid can be updated to their final variants